app/core/api/views.py

```diff
@@ -518,3 +520,9 @@ class ProjectUrlViewSet(viewsets.ModelViewSet):
     permission_classes = [IsAuthenticated]
     queryset = ProjectUrl.objects.all()
     serializer_class = ProjectUrlSerializer
+
+
+class ProjectStackElementXrefViewSet(viewsets.ModelViewSet):
+    queryset = ProjectStackElementXref.objects.all()
+    serializer_class = ProjectStackElementXrefSerializer
+    permission_classes = [IsAuthenticatedOrReadOnly]
```

Comment:
We decided to restrict everything to logged in users only for now.

```suggestion
    permission_classes = [IsAuthenticated]
    queryset = ProjectStackElementXref.objects.all()
    serializer_class = ProjectStackElementXrefSerializer
```